src/events/attributes.rs

```diff
@@ -791,6 +870,57 @@ mod xml {
     use super::*;
     use pretty_assertions::assert_eq;
 
+    #[test]
+    fn attribute_value_normalization() {
```

Comment:
It is better to have a `mod` with few test functions that tests different aspects of normalization. Then, if tests failed, we will known what is exactly wrong